nautobot/ipam/tests/test_api.py

```diff
@@ -154,25 +163,45 @@ def setUpTestData(cls):
             virtual_machine=cls.test_vm,
             rd="65000:4",
         )
+        VRFDeviceAssignment.objects.create(
+            vrf=cls.vrfs[0],
+            virtual_device_context=cls.vdcs[0],
+            rd="65000:5",
+        )
+        VRFDeviceAssignment.objects.create(
+            vrf=cls.vrfs[0],
+            virtual_device_context=cls.vdcs[1],
+            rd="65000:6",
+        )
 
         cls.create_data = [
             {
                 "vrf": cls.vrfs[2].pk,
                 "device": cls.devices[4].pk,
                 "virtual_machine": None,
-                "rd": "65000:4",
+                "virtual_device_context": None,
```

Comment:
I would hope that in the "create" API data we can just omit the null values, rather than needing to explicitly set them to None?